com.unity.ml-agents.extensions/Documentation~/Match3.md

```diff
@@ -0,0 +1,67 @@
+# Match-3 Game Support
```

Comment:
This document is not linked anywhere (not that I could see). I think we can link it from the examples page or from the com.unity.ml-agents.extensions.md page in the contents section

Reply:
Added links from the extensions readme